src/main/kotlin/com/cognifide/gradle/aem/tooling/tail/io/LogFiles.kt

```diff
@@ -0,0 +1,61 @@
+package com.cognifide.gradle.aem.tooling.tail.io
+
+import com.cognifide.gradle.aem.common.AemExtension
+import com.cognifide.gradle.aem.common.AemTask
+import com.cognifide.gradle.aem.common.Formats
+import com.cognifide.gradle.aem.tooling.tail.TailException
+import com.cognifide.gradle.aem.tooling.tasks.Tail
+import java.io.*
+import java.net.URI
+import java.net.URL
+import org.apache.commons.io.FileUtils
+import org.gradle.util.GFileUtils
+
+class LogFiles(private val aem: AemExtension) {
+
+    fun mainUri(instanceName: String) = uri(main(instanceName))
+
+    fun clearMain(instanceName: String) = main(instanceName).bufferedWriter().use { it.write("") }
+
+    fun clearSnapshots(instanceName: String) =
+            FileUtils.deleteDirectory(AemTask.temporaryDir(aem.project, Tail.NAME, "$instanceName/snapshots"))
+
+    fun writeToSnapshot(instanceName: String, writerBlock: (BufferedWriter) -> Unit): URI {
+        val snapshotFile = snapshot(instanceName)
+        snapshotFile.bufferedWriter().use(writerBlock)
+        return uri(snapshotFile)
+    }
+
+    fun writeToMain(instanceName: String, writerBlock: (FileWriter) -> Unit) =
+            FileWriter(main(instanceName).path, true).use(writerBlock)
+
+    private fun main(instanceName: String) =
+            AemTask.temporaryFile(aem.project, "${Tail.NAME}/$instanceName", "error.log")
```

Comment:
`error.log` should not be hardcoded; any AEM dev could introduce own error.log file with even TRACE level which we should be able to track so that even `logs/error.log` should not be hardcoded but be configurable

Reply:
OK